### ISSUE TITLE
Support changing app icon on Android

### ISSUE THREAD

**Issue by IchordeDionysos** [wal33d006/app_icon_changer#3]
It looks like it should be possible on Android by following this guide:
https://blog.jakelee.co.uk/programmatically-changing-app-icon/

**Comment by IchordeDionysos** [comment]
Otherwise, this should be possible:
https://stackoverflow.com/questions/4854197/how-can-i-place-app-icon-on-launcher-home-screen

**Comment by wal33d006** [comment]
The second link is regarding shortcut, which is not actually changing app icon. Regarding the first one, the article itself says it is not recommended for production use. Also, Android officially doesn't have any proper API exposed to actually change the icon; there are workarounds but I don't think they are good enough otherwise we would have an icon change option for GitHub's android app as well 🙌

**Comment by IchordeDionysos** [comment]
The second link is probably the option Instagram has opted for, for their anniversary easter egg:
https://www.theverge.com/2020/10/6/21503781/instagram-change-icons-birthday-celebration

**Comment by wal33d006** [comment]
Are you sure it is available for android as well? I have seen a lot of videos of people enjoying that feature, all of them were iPhone screen recordings.
If they have used it for Android as well, we can think of adding it here as well

**Comment by creativecreatorormaybenot** [comment]
@wal33d006 Yes, you can scroll up in settings in Instagram on Android at the moment.

This will allow you to create shortcuts for the same icons that you can select on iOS 👍

**Comment by wal33d006** [comment]
That's great! Let's add this to the plugin then. If you can, create a PR for this, otherwise I will park this item and will check it out over the weekend 👍
